packages/core/src/components/menu/_common.scss

```diff
@@ -85,20 +90,26 @@ $dark-menu-item-intent-colors: (
 
   &:active,
   &.#{$ns}-active {
-    background-color: $menu-item-color-active;
+    @include menu-item-active(false);
```

Comment:
Active state used to have this default value but it was always overridden by the selected modifier that I removed below. Including the mixin keep the state where no intent active item is rendered blue.